features/page_element_interaction.feature

```diff
@@ -44,6 +44,10 @@ Feature: Page element interaction
     When I navigate to the home page
     Then all expected elements are present
 
+  Scenario: Page with excluded_elements
```

Comment:
Optional Linting
As you're directly referencing a method inside a Scenario wrap it in backticks (Key left of 1), to show this